### ISSUE TITLE
Question: one or multiple status fields for consignment?

### ISSUE THREAD

**Issue by bmeesters** [opentripmodel/otm5-change-requests#43]
@BobZuidhoek proposes to replace the current status field with three fields for the different sub-processes a consignment can be in, see also:

* https://github.com/opentripmodel/otm5-change-requests/issues/39
* https://github.com/opentripmodel/otm5-change-requests/issues/40
* https://github.com/opentripmodel/otm5-change-requests/issues/41

The upside being that you get a more precise model. The downside being that there are multiple statuses you have to keep track off. There are also some questions on how the exact flow should work (e.g. is it mandatory to have the previous status filled when starting on the next?). So I want to zoom out a bit and question to overall approach. Do we want to extend the current `status` field on a consignment with the missing options, or do we want to split it up? If the answer is yes we can look at the detailed proposals of Bob in more detail.

@thomaskolmans, @gerardpeters, @rverberne, @JaapBraam do any of you have an opinion on this as users of OTM5. I know some of you go quite in depth with consignments. So is this a good addition or not?

**Comment by BobZuidhoek** [comment]
Good approach Bas to (also) take the helicopter view on this. 

Take into consideration that I have now suggested 3 status fields. I think when you start to add other functionality (e.g. around invoicing and payment) to the model, you'll likely find new main-status fields (or small models in itself) as well, for example: 

- InvoicingStatus (NotInvoiced, PartiallyInvoiced, CompletelyInvoiced, etc.). Maybe also other attributes within this sub-model such as InvoiceDisputed (boolean) and attribute InvoiceDisputedReason. And of course the invoice-lines itself, total amount, etc.  
- InvoicePaymentStatus (Outstanding, PartiallyPaid, CompletelyPaid, etc.). Maybe also other attributes within this submodel such as AmountPaid, AmountDue, etc. 

So also approach this as certain 'design route' that does not necessarily end with just the 3 status fields I suggested earlier.
